### ISSUE TITLE
Server instantly closes on startup

### ISSUE THREAD

**Issue by tryzombie501** [dotnetGame/MineCase#80]
I followed the install instructions it built without error however i have no idea why it closes because i can't read the error because it closes after printing it to the console

**Comment by sunnycase** [comment]
You should run `run_gateway.bat` after `run_server.bat` prints out `Press Ctrl+C to terminate...`.

**Comment by tryzombie501** [comment]
I did it closes instantly with out anything explaining whats happend

**Comment by Alinshans** [comment]
@tryzombie501 
Did you directly double-click the `run_server.bat`? Please wait a moment, and it will appear two console windows, you will see `Press Ctrl+C to terminate...` after some colorful output on the second windows. If you can see this line, double-click the `run_gateway.bat` after seeing that.

Which step did you encounter in this problem? Or what the last output can you see?

**Comment by tryzombie501** [comment]
I ran run_server.bat eventually it closed and dotnet.exe opens, it ran through some stuff and then i came across "Press Ctrl+C to terminate..." so i terminate the program via "Ctrl+C" and run "run_gateway.bat" a command prompt appears then closes, dotnet once again opens it remains black for a while then the console output is going so fast i can't read it. I tried stopping the console at the error but it always closes before i can stop it.

However the error is after this console output
"info: MineCase.Gateway.Program[0]
      Connecting to cluster..."

Seems like i'm getting some kind of exception output in the console but it closes before i can read it or stop it. Is there anyway i can stop it from closing so i can get a look at the error?
